docs/_data/modules.yaml

```diff
@@ -49,6 +49,7 @@ list:
   - segmented-buttons
   - select
   - signal
+  - skeleton-loader
```

Comment:
We should probably name the module "skeleton" to match the class, like we do everywhere else. Alternatively rename the class to match the module/file name.

Reply:
Sure, I will rename these to `skeleton` then.